advanced/editor-control-identifiers.md

```diff
@@ -151,6 +151,7 @@ Use the [toolbar]({{ site.baseurl }}/configure/editor-appearance/#toolbar) or th
 | undo                   | core                                                                                                                               | Edit                                                                                                     | To undo the last operation.                                                                               |
 | visualaid              | core                                                                                                                               | View                                                                                                     | Toggles visual aids on/off.                                                                               |
 | a11ycheck              | [a11ychecker]({{ site.baseurl }}/plugins/a11ychecker/)                                                                             | Tools                                                                                                    | Opens the accessibility checker dialog.                                                                   |
+| advtablesort           | [advtable]({{ site.baseurl }}/plugins/advtable/)                                                                                   | Table                                                                                                    | Table sort menu item with related controls.                                                               |
```

Comment:
I wonder if the description here should mention "advanced" so it's more obvious which plugin its from?

Reply:
Fixed :)